### ISSUE TITLE
Issue with harvest level config

### ISSUE THREAD

**Issue by ThuhElement** [tyra314/ToolProgression#60]
Hello,

Before I start, thank you for an awesome mod it has made my modpack so much easier to customize! I use this for so many things within the pack. I am working on a modpack inspired by Blood N Bones that will be very similar when I'm finished!

Issue-

I have Tinkers paired with Metallurgy 4: Reforged and I have had no issues with Metallurgy tools. I have 12 mining levels and they work flawless for Metallurgy. On the tinker side of things sadly I lose the balance in harvest level after a certain point. I've noticed that if both Tinkers and Metallurgy have the same material, the "material overwrite.cfg." will allow me to change the harvest level E.g. (Copper and Electrum) However, any exclusive metals to Metallurgy, the cfg will not overwrite and remains the default harvest level for tinkers variant of the pick.

So, If there is a way to have tinkers material variants of metallurgy metals adjustable to keep similar tool progression, that would be amazing.

Thank you

p.s I am sending a txt version of the materials.cfg and materials_overwrites.cfg for you.

[materials_overwrites.txt](https://github.com/tyra314/ToolProgression/files/3340729/materials_overwrites.txt)

[materials.txt](https://github.com/tyra314/ToolProgression/files/3340721/materials.txt)

**Comment by tyra314** [comment]
I don't know if you solved your issue and if I'm even understanding the issue, but the materials_overwrite.cfg has an error in it.

The last line should read 
```	
	S:adamantine=10
```

**Comment by ThuhElement** [comment]
Hey sorry for the confusion! I have been up for quite some time honing in the modpack. This issue was resolved and I didn't even understand when it had happened. I was working on another section of the modpack when I noticed all the tools working perfectly. A few things that changed though, I had installed CoFH mods and I noticed there was a "Txt" file alongside the overwrite.cfg and I left all the stats the same. 

The bottom of the list I was testing things to see if I could crack the code to why it wasn't working so I had left it without the "S:" lol. Eventually I went back and set all the tools properly and it worked like a charm. I wish I knew what I fixed so you'd be prepared to help someone else if they had the same problem. Nonetheless I was shocked and felt a huge thorn removed. This was one of the last issues I had before publishing my spiritual successor to Blood N Bones. Thank you for making this modpack so easy to configure despite my mishap! This pack would not even be possible without your mod!
